crates/ruff_workspace/src/options.rs

```diff
@@ -362,7 +362,6 @@ pub struct Options {
         line-length = 120
         "#
     )]
-    #[cfg_attr(feature = "schemars", schemars(range(min = 1, max = 320)))]
```

Comment:
FWICT, it looks like this is applying to the `Option<LineLength>`, rather than the `LineLength` inside the `Option`?

Reply:
FWIW, I can replicate it with a MWE:

```rust
use schemars::{schema_for, JsonSchema};

#[derive(JsonSchema)]
pub struct MyStruct {
    pub my_int: i32,
}

#[derive(JsonSchema)]
pub struct MyConfig {
    #[schemars(range(min = 1, max = 320))]
    my_struct: Option<MyStruct>,
}

fn main() {
    let schema = schema_for!(MyConfig);
    println!("{}", serde_json::to_string_pretty(&schema).unwrap());
}
```

And one fix is to move the location of the attribute:

```rust
use schemars::{schema_for, JsonSchema};

#[derive(JsonSchema)]
pub struct MyStruct {
    #[schemars(range(min = 1, max = 320))]
    pub my_int: i32,
}

#[derive(JsonSchema)]
pub struct MyConfig {
    my_struct: Option<MyStruct>,
}

fn main() {
    let schema = schema_for!(MyConfig);
    println!("{}", serde_json::to_string_pretty(&schema).unwrap());
}

```

But not sure how that works from the outer location, and not sure what `cfg_attrs` is. I see an `inner()`, but it seems to only apply to things like Vector, not to Enums.

Reply:
Ahh, I know what that is, and how to fix it (I think).